packages/date-picker/src/vaadin-lit-month-calendar.js

```diff
@@ -102,7 +102,7 @@ class MonthCalendar extends MonthCalendarMixin(ThemableMixin(PolylitMixin(LitEle
       this._weeks = this._getWeeks(this._days);
     }
 
-    if (props.has('month') || props.has('minDate') || props.has('maxDate')) {
+    if (props.has('month') || props.has('minDate') || props.has('maxDate') || props.has('isDateDisabled')) {
```

Comment:
Need to drop this change. We end up not needing the custom `isDateDisabled` check here because this should only check `min` and `max` to determine if we should disable the entire month.

Reply:
updated